bower.json

```diff
@@ -1,15 +1,14 @@
 {
   "name": "cfgov-sheer-templates",
   "description": "Sheer templates for consumerfinance.gov",
-  "version": "0.3.1",
+  "version": "1.0.0",
```

Comment:
Sorry, I should have written: *If bower relied on the `version` property, which it doesn't, then* if you listed 1.2.0 as the latest version in bower.json but there was no 1.2.0 tag at GH, `bower install cfgov-sheer-templates` would *not behave as expected*.

It's nice to have it in bower.json because it lets users easily check what the latest version is buuuuut it requires a maintenance burden of having to manually update it whenever we release a new version.